src/screens/GroupSetting/hooks/useGroupSetting.js

```diff
@@ -55,6 +63,7 @@ const useGroupSetting = ({navigation, route}) => {
       } else if (channel?.data?.image) {
         dataEdit.image = channel?.data?.image;
       }
+      console.log({dataEdit}, 'nusu');
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

Console logs should not be present in production code as they can expose sensitive information and clutter the console output. Please remove this line.
```diff
-   console.log({dataEdit}, 'nusu');
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->